python/api-sqs-lambda/README.md

```diff
@@ -0,0 +1,42 @@
+
+# API Gateway + SQS + Lambda
```

Comment:
Can you add a section at the end of this file, letting people know how they can interact with the APIGateway after the stack has been deployed and how to check the message was received by the Lambda function and view the contents of the message via the function's logs?

You could provide a sequence of AWS CLI commends with 1 line commentary on what each command is doing, similar to the steps you've used to describe the steps to activate virtualenv and `cdk synth`.

Reply:
Added testing instructions in the README file. Hope this helps!